launcher/ui/pages/instance/ExternalResourcesPage.cpp

```diff
@@ -248,6 +256,15 @@ bool ExternalResourcesPage::onSelectionChanged(const QModelIndex& current, const
     int row = sourceCurrent.row();
     Resource const& resource = m_model->at(row);
     ui->frame->updateWithResource(resource);
-
     return true;
 }
+
+QString ExternalResourcesPage::extraHeaderInfoString()
+{
+    if (ui && ui->treeView && ui->treeView->selectionModel()) {
+        auto selection = m_filterModel->mapSelectionToSource(ui->treeView->selectionModel()->selection()).indexes();
+        if (auto count = std::count_if(selection.cbegin(), selection.cend(), [](auto v) { return v.column() == 0; }); count != 0)
+            return tr("[%1 installed, %2 selected]").arg(m_model->size()).arg(count);
```

Comment:
```suggestion
            return tr(" [%1 installed, %2 selected]").arg(m_model->size()).arg(count);
```
this looks a little cramped imo
![image](https://github.com/PrismLauncher/PrismLauncher/assets/48872998/14eec71f-d73c-4923-ac31-83097d98e9bc)

Reply:
I think rounded brackets would be better